src/domain/hanke/hankeView/HankeView.test.tsx

```diff
@@ -21,7 +21,7 @@ test('Draft state notification is rendered when hanke is in draft state', async
   render(<HankeViewContainer hankeTunnus="HAI22-1" />);
 
   await waitForLoadingToFinish();
-
+  screen.debug(undefined, 1000000);
```

Comment:
Should this be removed?

Reply:
Yes, it should. Thank you for noticing and sorry about the forget.

Reply:
No problem.